options/locale/locale_en-US.ini

```diff
@@ -2138,6 +2138,11 @@ settings = Settings
 settings.desc = Settings is where you can manage the settings for the repository
 settings.options = Repository
 settings.public_access = Public Access
+settings.public_access_desc = Configure public access permissions for each units.
+settings.public_access.docs.not_set = "Not set" means there is no extra public access rules for this repo, it follows its own visibility and member permissions.
+settings.public_access.docs.anonymous_read = "Anonymous read" means non-signed-in users can access the unit.
```

Comment:
Using quotes will definitely cause bugs (due to the buggy ini system)

![image](https://github.com/user-attachments/assets/511ef019-9f0e-45d5-a994-ca42680e2d67)

Reply:
Will think about how to handle these cases.